index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/font-awesome.min.css" />
+  </head>
+
+  <body>
+    <div class="nav-bar">
+      <h2>TEXT</h2>
+      <nav id="navbar">
+        <ul>
+          <li><a href="#welcome-section"> Home</a></li>
+          <li><a href="#projects"> Projects</a></li>
+          <li><a href="#contacts"> Contact me</a></li>
+        </ul>
+      </nav>
+    </div>
```

Comment:
It would be better if instead of using <div> tag you use <header> one, as this gives better semantic value to your HTML.